launch/prefix_tf_republisher.launch.yaml

```diff
@@ -0,0 +1,21 @@
+launch:
+- arg:
+    name: namespace
+    default: ""
+
+- node:
+    pkg: "hector_multi_robot_tools"
+    exec: "prefix_tf_republisher"
+    name: "tf_republisher"
+    namespace: "$(var namespace)"
+
+- node:
+    pkg: "hector_multi_robot_tools"
+    exec: "prefix_tf_republisher"
+    name: "tf_static_republisher"
+    namespace: "$(var namespace)"
+    remap:
+        - from: "tf"
+          to: "tf_static"
+        - from: "/tf"
+          to: "/tf_static"
```

Comment:
This does not look intentional.

Reply:
I will rename the input and output topics to `input_tf_topic` and `output_tf_topic`, so the remappings are more readable.